frodo.js

```diff
@@ -4,63 +4,69 @@
  * This plugin performs initialization (like calling mermaid.initialize())
```

Comment:
SCRIPT-This block of code could benefit from some comments explaining the logic behind it.